### PR TITLE
Add several misspell words

### PR DESCRIPTION
**Pull request by vikivivi**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -3213,17 +3213,34 @@ bacground->background
 bacic->basic
 backards->backwards
 backbround->background
+backbrounds->backgrounds
+backedn->backend
+backedns->backends
+backened->backend
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:3219]: Suggest blackened here too, probably first

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -3213,17 +3213,34 @@ bacground->background
...
+backeneds->backends
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:3220]: Suggest blackens here too, possibly first

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -3287,8 +3306,11 @@ basci->basic
 bascially->basically
 bascktrack->backtrack
 basf->base
+basicallly->basically
 basicaly->basically
 basicly->basically
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:3311]: Would you mind adding `basiclly->basically` too?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -3582,11 +3604,13 @@ bloaded->bloated
 blocack->blockack
 bloccks->blocks
 blocekd->blocked
+blockhain->blockchain
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:3607]: Also add the plural version of this typo too `blockhains`

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -3582,11 +3604,13 @@ bloaded->bloated
...
 blockin->blocking
 bloddy->bloody
 blodk->block
 blohted->bloated
 blokcer->blocker
+blokchain->blockchain
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:3613]: And this one.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -16654,9 +16722,11 @@ midified->modified
 migrateable->migratable
 migt->might, midget,
 migth->might
+miinimum->minimum
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:16725]: Can you add the -ise, -ize, -ised, -ized, -ising, -izing, -isation and -ization versions of this typo too.

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -16728,6 +16798,10 @@ minimium->minimum
 minimumm->minimum
 minimumn->minimum
 minimun->minimum
+minimze->minimize
+minimzed->minimized
+minimzes->minimizes
+minimzing->minimizing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:16804]: Can you add the British English (is) versions of this typos too.

**Comment by vikivivi** [comment]
Done. Beside `blackened` and `blackens` been the first. Friends of `blockchain`, `backend` and `typecast` are also first entries.

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (peternewman) [reply codespell_lib/data/dictionary.txt:3312]: Done.

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -4556,11 +4614,13 @@ characteters->characters
 charactetistic->characteristic
 charactetistics->characteristics
 charactetr->characters
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4616]: This should correct to the singular.

File: codespell_lib/data/dictionary.txt [hunk 9]
```diff
@@ -4556,11 +4614,13 @@ characteters->characters
...
-charactetrs->characterss
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4559]: Do you want to add the typo fix here as a misspelling itself `characterss`

**Pull request merged**